commit 88b3eaa499ddbe5a36c34840117563bd54ec4ec2
Author: nohee ko <noheek@broadcom.com>
Date:   Mon Oct 11 18:33:45 2010 -0700

    staging: brcm80211: remove duplicated function, wf_channel2mhz
    
    remove duplicated function, wf_channel2mhz
    
    Signed-off-by: Nohee Ko <noheek@broadcom.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

--- a/drivers/staging/brcm80211/util/bcmwifi.c
+++ b/drivers/staging/brcm80211/util/bcmwifi.c
@@ -176,22 +176,6 @@ int wf_mhz2channel(uint freq, uint start
  *
  * Reference 802.11 REVma, section 17.3.8.3, and 802.11B section 18.4.6.2
  */
-#ifdef BRCM_FULLMAC
-int wf_channel2mhz(uint ch, uint start_factor)
-{
-	int freq;
-
-	if ((start_factor == WF_CHAN_FACTOR_2_4_G && (ch < 1 || ch > 14)) ||
-	(ch <= 200))
-		freq = -1;
-	if ((start_factor == WF_CHAN_FACTOR_2_4_G) && (ch == 14))
-		freq = 2484;
-	else
-		freq = ch * 5 + start_factor / 2;
-
-	return freq;
-}
-#else /* !BRCM_FULLMAC */
 int wf_channel2mhz(uint ch, uint start_factor)
 {
 	int freq;
@@ -206,4 +190,3 @@ int wf_channel2mhz(uint ch, uint start_f
 
 	return freq;
 }
-#endif /* BRCM_FULLMAC */
